src/meson.build

```diff
@@ -0,0 +1,67 @@
+sources = files(
+  'buildable.vala',
+  'builder.vala',
+  'error.vala',
+  'libyaml-glib.vala',
+  'loader.vala',
+  'writer.vala',
+)
+
+deps = [
+  gobject_introspection_dep,
+  yaml_dep,
+  gmodule_dep,
+]
+
+revision = 0
+current = 1
+age = 0
+current_minus_age = current - age
+project_libversion = '@0@.@1@.@2@'.format(current_minus_age, age, revision)
+
+libglib_yaml = shared_library(
+  '@0@-@1@'.format(project_name, major_version),
+  version: project_libversion,
+  sources: sources,
+  dependencies: deps,
+  install: true,
```

Comment:
Installation is missing the header and the VAPI. You can use `install_dir: [true, true, true]` feature to do that.

More info here: https://mesonbuild.com/Vala.html

Reply:
sure, let me rework that. gitg should be a good guide as it have libraries in vala too.

Reply:
done